cmd/eventlistener/main.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+)
+
+func handler(w http.ResponseWriter, r *http.Request) {
+	log.Printf("EventListener Pod received a request: %+v", r)
+	fmt.Fprintf(w, "Hello from the EventListener Pod! \n")
+}
+
+func main() {
+	log.Print("EventListener Pod started")
+	log.Printf("Listen and serve on port 8082")
+
+	http.HandleFunc("/", handler)
+	log.Fatal(http.ListenAndServe(":8082", nil))
+}
```

Comment:
late to the party but: @ncskier in a follow up PR could we add a README for the event listener binary? sometimes it can be hard to get up to speed on what all the stuff in `cmd` is for when you're new to it